src/configure/helper/remoteServiceUrlHelper.ts

```diff
@@ -18,6 +18,14 @@ export class RemoteServiceUrlHelper {
     public static provisioningServiceRedirectUrl: string = "https://go.microsoft.com/fwlink/?linkid=2142042";
 
     public static async getTemplateServiceDefinition(): Promise<IServiceUrlDefinition> {
+        const deployment = process.env["DEPLOY_TO_AZURE_ENVIRONMENT"];
+        if (deployment != undefined && deployment === "development") {
```

Comment:
Is there a way to make this check more stricter? I mean is there a way to let this happen only when extension is running locally and not via vsix.
I am thinking from security point of view and it could open up a way for a bad user to make the flow go to a spoofed url and dupe a good user

Reply:
There are chances where we might be taking extension from market place and setting env in our local and using to test to locally deployed service. Then we will not be able differentiate. To avoid we have env name DEPLOY_TO_AZURE_EXT_ENVIRONMENT as unique as possible